src/views/components/topology/topo.vue

```diff
@@ -305,7 +306,7 @@ export default {
   },
     toggleNode(nodeCircle, currNode, isHover) {
     if (isHover) {
-        // 提升节点层级 
+        // ?????????????????? 
```

Comment:
Remove question mark and Chinese comments.

Reply:
Sorry, My editor dosen't  set encoding. 
I remove directly the comments?

Reply:
Yes.